tests/MMS/advection/runtest

```diff
@@ -32,7 +34,8 @@ options = [
     ]
 
 # List of NX values to use
-nxlist = [16, 32, 64, 128, 256, 512, 1024]
+#[16, 32, 64, 128, 256, 512, 1024]
+nxlist = [64, 128]
```

Comment:
I'm not sure about this, I don't think two points are enough for an MMS test

Reply:
I haven't found a single test that takes more than two points to evaluate to success condition.

Basically all use ` order = log(error_2[-1] / error_2[-2]) / log(dx[-1] / dx[-2])`.

Reply:
### Arakawa

nx | time | order
--|--|--
 16 |   2.410 s | ?
 32 |   3.438 s | 3.255013
 64 |  10.307 s | 2.490309
 128 |  21.605 s | 2.060740
 256 |  45.281 s | 2.004003
 512 | 104.097 s | 1.998664

### U1

nx | time | order
--|--|--
 16 |   2.531 s | ?
 32 |   5.458 s | 2.373164
 64 |   9.895 s | 1.242753
 128 |  18.549 s | 0.972551
 256 |  49.725 s | 0.971749
 512 | 138.652 s | 0.985141


### Central

nx | time | order
--|--|--
 16 |   2.737 s | ?
 32 |   3.666 s | 3.449089
 64 |  14.249 s | 2.902876
 128 |  28.197 s | 2.220268
 256 |  47.013 s | 2.051120
 512 | 126.226 s | 2.015397

### Weno 3

nx | time | order
--|--|--
 16 |   5.916 s | ?
 32 |  15.774 s | 3.480261
 64 |  29.917 s | 3.135221
 128 |  41.231 s | 2.400913
 256 |  58.130 s | 2.106673
 512 | 140.946 s | 2.038875